src/amp/components/AdConsent.tsx

```diff
@@ -4,9 +4,11 @@ import { JsonScript } from './JsonScript';
 const sourcepointDomain = 'sourcepoint.theguardian.com';
 
 const pubData = {
+	authId: 'CLIENT_ID',
+	client_id: 'CLIENT_ID',
 	// Matches ampViewId from https://ophan.theguardian.com/amp.json
-	pageViewId: 'PAGE_VIEW_ID_64',
-	browserId: 'CLIENT_ID',
+	page_view_id: 'PAGE_VIEW_ID',
+	page_view_id_64: 'PAGE_VIEW_ID_64',
```

Comment:
This will require unique JSON deserialisation, annoyingly.

The data processing expects a key with proper camelCase `pageViewId`.